packages/cli/lib/configs.js

```diff
@@ -0,0 +1,53 @@
+const request = require('request-promise-native');
+const { logger } = require('@hubspot/cli-lib/logger');
+const { logErrorInstance } = require('@hubspot/cli-lib/errorHandlers');
+
+// https://developer.github.com/v3/#user-agent-required
+const USER_AGENT_HEADERS = { 'User-Agent': 'HubSpot/hubspot-cms-tools' };
```

Comment:
I think we should create a constant in `packages/cli-lib/http/requestOptions` for the constant and share it between here and `getRequestOptions`.

Reply:
Nice point. Created DEFAULT_USER_AGENT_HEADERS constant, and shared it to projects and github.js